dialect/main.py

```diff
@@ -125,6 +125,8 @@ def setup_actions(self):
         self.set_accels_for_action('win.forward', ['<Alt>Right'])
         self.set_accels_for_action('win.switch', ['<Primary>S'])
         self.set_accels_for_action('win.clear', ['<Primary>D'])
+        self.set_accels_for_action('win.inc-font', ['<Primary>plus', 'KP_Add'])
+        self.set_accels_for_action('win.dec-font', ['<Primary>minus', 'KP_Subtract'])
```

Comment:
We also want the modifier for the num pad keys.
```suggestion
        self.set_accels_for_action('win.inc-font', ['<Primary>plus', '<Primary>KP_Add'])
        self.set_accels_for_action('win.dec-font', ['<Primary>minus', '<Primary>KP_Subtract'])
```